src/applications/gi/components/profile/CautionaryInformation.jsx

```diff
@@ -121,8 +122,12 @@ export class CautionaryInformation extends React.Component {
 
     const allComplaints = complaints.pop();
 
+    const cautionTableFix = environment.isProduction()
```

Comment:
Why not push the fix out as soon as it's ready?

Reply:
We want to test the fix in staging before the fix going out to production.